### DISCUSSION TITLE
Where can I download Java UDF for v1.2.7?

### DISCUSSION THREAD

**Discussion by sherlockhoatszx** [apache/doris#26665] [A - General / Q&A]
1.2.3之前下载的时候都有udf jar，1.2.7的时候只提供了安装说明。没有提供dependencies的jar，我下了1.2.0的jar 提示依赖有问题。无法启动

**Comment by LemonLiTree** [comment]
里面已经自带了

**Comment by sherlockhoatszx** [comment]
那官网的教程 里面的这一步应该标注出来1.2.4以上不需要这一步。安装 Java UDF 函数
SinceVersion 1.2.0安装Java UDF 函数因为从1.2 版本开始支持Java UDF 函数，需要从官网下载 Java UDF 函数的 JAR 包放到 BE 的 lib 目录下，否则可能会启动失败。

**Reply by ixzc** [reply]
Ok, we'll fix the document

**Reply by wdqzdd** [reply]
到现在还没修改呢，按照官方文档，掉这个坑里了。

**Reply by Hanchers** [reply]
now, it  does not fix  yet

**Comment by heyline** [comment]
官方文档写的太智障了
